frontend/sw360-portlet/src/main/webapp/html/homepage/myprojects/view.jsp

```diff
@@ -1,5 +1,6 @@
 <%--
   ~ Copyright Siemens AG, 2013-2015. Part of the SW360 Portal Project.
+  ~ With contributions by Bosch Software Innovations GmbH, 2016.
```

Comment:
Not to denigrate your work in any way, but it seems like this file has no contributions by Bosch yet :-)